#!/usr/bin/env python


from dynamic_reconfigure.parameter_generator_catkin import ParameterGenerator, int_t

gen = ParameterGenerator()

gen.add("inflation_radius", int_t, 0, "The radius in meters to which the map inflates obstacle cost values.", 8, 0, 10)

exit(gen.generate("autonomous_nav", "potential_map", "Auto"))